sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -43,6 +57,83 @@ export type AnalyzeActionParameters<ActionName extends AnalyzeActionName> = {
     LanguageDetection: LanguageDetectionAction;
 }[ActionName];
 
+// @public
+export type AnalyzeBatchAction = {
```

Comment:
Can `kind` be worked into the actions themselves and then this can be a simple union of named interfaces? This type is really gnarly and will be unreadable on docs.ms.

Reply:
Good point. I was trying to avoid introducing too many named things but yeah the docs experience is important too. I broke that type into a union of individual ones, how does it look now?